goldenmaster/Plugins/TbSimple/Source/TbSimple/Private/Tests/TbSimpleSimpleArrayInterfaceTests.cpp

```diff
@@ -40,6 +40,26 @@ bool UTbSimpleSimpleArrayInterfaceImplementationPropertyPropIntTest::RunTest(con
 	return true;
 }
 
+IMPLEMENT_SIMPLE_AUTOMATION_TEST(UTbSimpleSimpleArrayInterfaceImplementationPropertyPropInt32Test, "TbSimple.SimpleArrayInterface.Implementation.Property.PropInt32", EAutomationTestFlags::ApplicationContextMask | EAutomationTestFlags::SmokeFilter)
+bool UTbSimpleSimpleArrayInterfaceImplementationPropertyPropInt32Test::RunTest(const FString& Parameters)
+{
+	// Do implement test here
```

Comment:
Maybe  "// Implement test here" instead ? Or what's the exact purpose of the comment?

Reply:
It is meant for the user of the generated code to add the test logic in this function.
Don't think we need the "Do" here, but would not start a new PR/commit just to remove it now. Ok?